content/posts/2023-11-20-tools-for-learning-llvm-tablegen.md

```diff
@@ -0,0 +1,433 @@
+---
+author: "David Spickett"
+date: "2023-11-20"
+tags: ["TableGen"]
+title: "Tools for Learning LLVM TableGen"
+---
+
+[TableGen](https://github.com/llvm/llvm-project/tree/main/llvm/utils/TableGen)
+is a language used within the LLVM project for generating all kinds of files,
+when maintaining those files by hand would be very difficult.
```

Comment:
is a language we use in the LLVM project to generate a variety of files - when manual maintenance would be difficult.

Reply:
Done.